internal/cli/clients/client.go

```diff
@@ -177,6 +319,10 @@ func (c *EpinioClient) ServicePlanMatching(ctx context.Context, serviceClassName
 
 	result := []string{}
 
+	// TODO Create and use server endpoints. Maybe use existing
```

Comment:
Ah nice, these TODOs are for
https://github.com/epinio/epinio/issues/508

Reply:
Yes